src/Fetcher/fetcher.go

```diff
@@ -1,12 +1,15 @@
 package Fetcher
 
 import (
+	"fmt"
 	"io/fs"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strings"
+
+	"github.com/NovoNordisk-OpenSource/decentralized-tech-radar/Verifier"
 )
 
 type Repo struct {
```

Comment:
I assume we are removing this when we are merging into main?

Reply:
Why? The verifier is supposed to run when fetching or merging. If you remove this you can no longer do that

Reply:
Or are you talking about the repo struct? Think the line selection is a bit funky on GitHub mobile?